tests/config/bindings/tests.yml

```diff
@@ -1,13 +1,16 @@
-# the config map is passed in a metadata map to the Invoke request for output bindings
-componentType: output-binding
+componentType: bindings
 components:
   - component: redis
     operations: ["create", "operations"]
     config:
-      key: $((uuid))
+      output_key: $((uuid))
+      readBindingTimeout: 60
   - component: azure.blobstorage
     operations: ["create", "operations", "get"]
     config:
-      blobName: $((uuid))
+      output_blobName: $((uuid))
+      readBindingTimeout: 60
   - component: azure.storagequeues
+    operations: ["create", "operations", "read"]
```

Comment:
is the default readBindingTimeout sufficient for this?

Reply:
So far it's been sufficient, we could probably set it lower if we wanted to as it has always been pretty much instant for me.

Reply:
It has been very quick for me, so we could even make it lower if we wanted too.